src/Util/Api.php

```diff
@@ -95,6 +98,30 @@ public function getLang(): string {
 		return $this->lang;
 	}
 
+	/**
+	 * Get the on-wiki configuration from the `MediaWiki:WS_Export.json` page. Cached for 1 month.
+	 * @return mixed[] With keys: defaultFont
+	 */
+	public function getWikiConfig(): array {
```

Comment:
I feel like this methods belongs in its own service. Something like `WikisourceConfig` or `RemoteConfig` perhaps? What do you think?

Also, should we protect against `getDomainName()` or `getLang()` being empty?

Reply:
Good idea, then that service can take care of the structure of the page too, instead of having it spread out wherever it's used. I'll move it.